### PR TITLE
First Upload

### PR DESCRIPTION
**Pull request by jialinl6**

### CODE REVIEW

**Comment by ctessum** [comment]
You can review the contributor license agreement above, and if you agree with it, just type "I have reviewed and hereby sign the CLA" in the comment box below. It just basically just says that you agree that we can release your work as open source software.

File: src/SuperFast.jl [hunk 1]
```diff
@@ -0,0 +1,1931 @@
...
+
+using Markdown
+using InteractiveUtils
+
+# ╔═╡ 6cbe3c3e-4831-11ec-35e5-976f483e39f5
+using Catalyst, OrdinaryDiffEq, StochasticDiffEq,DiffEqBase,Plots
+
+# ╔═╡ adbe3fb1-fa14-4a2a-8263-daa606476f26
+md"""
+### Building Model
```
Comment (ctessum) [inline src/SuperFast.jl:12]: Could you add some background information here, such as what this algorithm does, what paper(s) it comes from, and any information other people might need to understand what you did here or use it in their own work?

**Review by ctessum — CHANGES_REQUESTED** [review]
Looking good! I added a couple of comments/suggestions.

**Comment by ctessum** [comment]
Also, you can review the contributor license agreement above, and if you agree with it, just type "I have reviewed and hereby sign the CLA" in the comment box below. It just basically just says that you agree that we can release your work as open source software.

**Comment by jialinl6** [comment]
> I have reviewed and hereby sign the CLA

File: src/SuperFast.jl [hunk 2]
```diff
@@ -0,0 +1,1931 @@
...
+"""
+
+# ╔═╡ 90c0a52b-eeeb-49fc-a610-847b5d470fa1
+@parameters r1 r2 r3 r6 r7 r9 r11 r12 r13a r13b r14 r15 r17 r21a r21c r22 rr1 rr2 rr3 rr4 rr5 rr6  r4 r5 r10 t
```
[the remark this answers is not in the document]
Comment (jialinl6) [reply src/SuperFast.jl]: Sure, I just added the units

File: src/SuperFast.jl (hunk 1 above)
Comment (jialinl6) [reply src/SuperFast.jl:22]: Thank you for the advice! I just added a brief description at the beginning.

**Comment by jialinl6** [comment]
recheck

**Comment by ctessum** [comment]
I have reviewed and hereby sign the CLA

**Comment by ctessum** [comment]
recheck

**Review by ctessum — APPROVED** [review]
Looks good, thanks!

**Pull request merged**